### PR TITLE
Add in Val's text for connectivity matrices

### PR DESCRIPTION
**Pull request by mattcieslak**
This adds in the text that @valeriejill tried to add the qsiprep docs. I also added some text for DSI Studio and moved the connectivity matrix section to the outputs page.

### CODE REVIEW

File: docs/builtin_workflows.rst [hunk 1]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
 
 \* Not recommended
 
+.. _connectivity_matrices:
+
+#####################
+Connectivity matrices
+#####################
```
Comment (tsalo) [inline docs/builtin_workflows.rst:517]:
```suggestion
*********************
Connectivity matrices
*********************
```

The section labeling needs to be adjusted. Here's the convention we have in conf.py.

````
# with overline, for parts
* with overline, for chapters
= for sections
- for subsections
^ for subsubsections
" for paragraphs
```

File: docs/builtin_workflows.rst [hunk 2]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
...
+additional comparisons.
+
+To skip this step in your workflow, you can modify an existing recon pipeline by removing the
+``action: connectivity`` section from the yaml file.
+
+.. _connectivity_atlases:
+
+*******
+Atlases
+*******
```
Comment (tsalo) [inline docs/builtin_workflows.rst:534]:
```suggestion
Atlases
=======
```

File: docs/builtin_workflows.rst [hunk 3]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
...
+                  sub-<label>_space-T1w_seg-<label>_dseg.nii.gz
+                  sub-<label>_space-T1w_seg-<label>_dseg.mif.gz
+                  sub-<label>_space-T1w_seg-<label>_dseg.json
+                  sub-<label>_space-T1w_seg-<label>_dseg.tsv
+
+
+.. _custom_atlases:
+
+Using custom atlases
+^^^^^^^^^^^^^^^^^^^^
```
Comment (tsalo) [inline docs/builtin_workflows.rst:598]:
```suggestion
Using custom atlases
--------------------
```

File: docs/builtin_workflows.rst [hunk 4]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
...
+The nifti images should be registered to the
+`MNI152NLin2009cAsym <https://github.com/PennLINC/qsirecon/blob/main/qsirecon/data/mni_1mm_t1w_lps.nii.gz>`_
+included in ``qsirecon``.
+It is essential that your images are in the LPS+ orientation and have the sform zeroed-out in the header.
+**Be sure to check for alignment and orientation** in your outputs.
+
+
+*********************
+Connectivity Measures
+*********************
```
Comment (tsalo) [inline docs/builtin_workflows.rst:612]:
```suggestion
Connectivity Measures
=====================
```

File: docs/builtin_workflows.rst [hunk 5]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
...
+.. code-block::
+
+   command                                # The command that was run
+   atlas_<atlas>_region_ids               # The region ids for the atlas (1 x n_parcels array)
+   atlas_<atlas>_region_labels            # The region labels for the atlas (1 x n_parcels array)
+   atlas_<atlas>_<measure>_connectivity   # The connectivity matrix for the atlas and measure (n_parcels x n_parcels array)
+
+
+MRtrix3 Connectivity Measures
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (tsalo) [inline docs/builtin_workflows.rst:649]:
```suggestion
-----------------------------
```

File: docs/builtin_workflows.rst [hunk 6]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
...
+   * *sift.radiusN.meanlength.connectivity*: a matrix containing mean length of sifted output
+
+The number ``N`` in ``radiusN`` indicates how many mm the algorithm would search up from a
+given streamline's endpoint for a cortical region. E.g., a radius of 2 indicates that
+if a streamline ended before hitting gray matter, the search for a cortical
+termination region could be up to 2mm from the endpoint.
+
+
+DSI Studio Connectivity Measures
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (tsalo) [inline docs/builtin_workflows.rst:666]:
```suggestion
--------------------------------
```

File: docs/builtin_workflows.rst [hunk 7]
```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
...
+MRtrix3 Connectivity Measures
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+MRtrix3 connectivity workflows produce 4 structural connectome outputs for each atlas.
+The 4 connectivity matrix outputs are
+
+   * *radiusN.count.connectivity*: raw streamline count based matrix
+   * *sift.radiusN.count.connectivity*: sift-weighted streamline count based matrix
+   * *radiusN.meanlength.connectivity*: a matrix containing mean length of raw streamlines
+   * *sift.radiusN.meanlength.connectivity*: a matrix containing mean length of sifted output
```
Comment (tsalo) [inline docs/builtin_workflows.rst:657]:
```suggestion
   * *atlas_<atlas>_radius<N>.count.connectivity*: raw streamline count based matrix
   * *atlas_<atlas>_sift.radius<N>.count.connectivity*: sift-weighted streamline count based matrix
   * *atlas_<atlas>_radius<N>.meanlength.connectivity*: a matrix containing mean length of raw streamlines
   * *atlas_<atlas>_sift.radius<N>.meanlength.connectivity*: a matrix containing mean length of sifted output
```

I think that's what this should be, but I could be wrong.

Comment (mattcieslak) [reply docs/builtin_workflows.rst:657]: I think you're right

File: docs/builtin_workflows.rst [hunk 8]
```diff
@@ -510,6 +510,178 @@ the model-fitting workflows and which sampling schemes work with them.
 
 \* Not recommended
 
+.. _connectivity_matrices:
+
+*********************
+Connectivity matrices
+*********************
```
Comment (tsalo) [inline docs/builtin_workflows.rst:517]: I just realized where you put this. Should this go in `outputs`?

File: docs/builtin_workflows.rst [hunk 9]
```diff
@@ -510,6 +510,178 @@ the model-fitting workflows and which sampling schemes work with them.
...
+
+Connectivity Measures
+=====================
+
+Connectivity measures are bundled together in binary ``.mat`` files,
+rather than as atlas- and measure-specific tabular files.
+
+.. warning::
+
+   We ultimately plan to organize the connectivity matrices accoring to the BIDS-Connectivity BEP,
```
Comment (tsalo) [inline docs/builtin_workflows.rst:617]:
```suggestion
   We ultimately plan to organize the connectivity matrices according to the BIDS-Connectivity BEP,
```

Just noticed my own typo

File: docs/builtin_workflows.rst (hunk 8 above)
Comment (mattcieslak) [reply docs/builtin_workflows.rst:517]: It's kind of outside of individual workflows (they all make these matfiles), but is slightly different for different workflows (the variables are named differently for dsi studio and mrtrix3)

**Review by tsalo — APPROVED** [review]

**Pull request merged**